README.md

```diff
@@ -9,35 +9,41 @@ Like what you see? :eyes: Give us a GitHub Star! :star:
 
 [![Focalboard](website/site/static/img/hero.jpg)](https://www.focalboard.com)
 
-[Focalboard](https://www.focalboard.com) is an open source, self-hosted alternative to Trello, Notion, and Asana.
+[Focalboard](https://www.focalboard.com) is an open source, self-hosted project management tool that's an alternative to Trello, Notion, and Asana.
 
-It helps define, organize, track and manage work across individuals and teams. Focalboard comes in two editions:
+It helps define, organize, track and manage work across individuals and teams. Focalboard comes in two main editions:
 
-* **Focalboard Personal Desktop**: A stand-alone desktop app for your todos and personal projects. This is a single-tenant locally run server running Focalboard for optimal speed and performance.
+* **[Personal Desktop](https://www.focalboard.com/download/personal-edition/desktop/)**: A stand-alone single-user Mac, Windows, or Linux desktop app for your todos and personal projects.
 
-* **Focalboard Personal Server**: A self-hosted server for your team to collaborate.
+* **[Mattermost Boards](https://www.focalboard.com/download/mattermost/)**: A self-hosted or cloud server for your team to plan and collaborate.
 
-The same MIT-licensed binary powers both desktop and server editions.
+Focalboard can also be installed as a standalone [personal server](https://www.focalboard.com/download/personal-edition/ubuntu/) for development and personal use.
 
 ## Try out Focalboard
 
-**Focalboard Personal Desktop (Windows, Mac or Linux Desktop)**
+### Focalboard Personal Desktop (Windows, Mac or Linux Desktop)
 
 Try out **Focalboard Personal Desktop** by going to the Windows Store or the Apple AppStore, searching for `Focalboard` and installing to run the compiled version locally.
 
 If you're running a Linux Desktop, [download the latest `focalboard-linux.tar.gz` release](https://github.com/mattermost/focalboard/releases), unpack the `.tar.gz` archive, and open `focalboard-app` from the `focalboard-app` folder.
 
 Note: For Windows and Mac users, while we don't yet offer **Focalboard Personal Desktop** outside of Store-based installs, it is in [consideration for the future](https://github.com/mattermost/focalboard/issues/99) (please upvote the ticket if you're interested in this addition).
 
-**Focalboard Personal Server (Ubuntu)**
+### Mattermost Boards
+
+Mattermost Boards combines project management tools with messaging and collaboration for teams of all sizes. To access and use Boards, [install or upgrade to Mattermost v6.0 or later](https://mattermost.com/get-started/?utm_source=focalboard&utm_campaign=focalboard). Then, select the menu in the top left corner of Mattermost and choose **Boards**.
```

Comment:
There may be a leap required to go from the page linked (https://mattermost.com/get-started/?utm_source=focalboard&utm_campaign=focalboard) to having Mattermost v6 + Boards up and running in order to "select the menu in the top left corner".  I think it would be nice to have a few more lines about selecting "Free Trial", installing, and then looking for the menu.

Reply:
Good point. This is something we will need to refine into fewer steps, but I've broken this out into a link to the self-managed deployment guide and Cloud landing page for now.